dex-it/src/test/scala/com/wavesplatform/it/sync/orderV3/PercentModeMatcherTestSuite.scala

```diff
@@ -0,0 +1,346 @@
+package com.wavesplatform.it.sync.orderV3
```

Comment:
I suggest to check also fee if order was partially filled, cancelled
Also what will happen, if we have order in Waves-Usd pair, fee is less than one percent, and total amount of price asset is less than 100? What if order was partiall filled twice, fee of both was less than one percent?